cli/azd/pkg/commands/pipeline/pipeline_manager.go

```diff
@@ -207,7 +207,7 @@ func (manager *PipelineManager) Configure(ctx context.Context) error {
 	var credentials json.RawMessage = nil
 	var err error
 
-	existingSP := os.Getenv("AZD_EXISTING_SP")
+	existingSP := os.Getenv("AZURE_EXISTING_SP")
```

Comment:
What does EXISTING_SP represent?   Do you have note/doc on it?

Reply:
It's a shim to allow for the use of an existing Service Principal.